File: StepByStep/Step 08/5622.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    string str;
    cin >> str;

    vector<string> dial = {{"-1"}, {"-1"}, {"-1"}, {"ABC"}, {"DEF"}, {"GHI"},
                           {"JKL"}, {"MNO"}, {"PQRS"}, {"TUV"}, {"WXYZ"}};

    int sec = 0;
    for(int i=0; str[i]; i++) {
        for(int j=0; j<dial.size(); j++) {
            if(dial[j].find(str[i]) != string::npos) sec += j;
        }
    }
    cout << sec;
    return 0;
}

File: StepByStep/Python/Step_06_문자열/5622.py
dial = ['ABC', 'DEF', 'GHI', 'JKL', 'MNO', 'PQRS', 'TUV', 'WXYZ']

words = input()
sec = 0
for i in words:
    for j in range(8):
        if dial[j].find(i) >= 0:
            sec += j+3

print(sec)



